feugy/atelier
# feat(plugin)!: framework-agnostic plugin

### What's in there?

In the event of supporting multiple UI framework, like React or Vue, it would be great to have a single vite-plugin.

This PR should allow this, by dynamically loading the UI bindings when starting vite, and delegating it the act of building the workframe content.

Are included here:

- [x] feat(plugin)!: renames as `vite-plugin-atelier`
- [x] feat(plugin): makes framework agnostic by delegating workframe content to framework bindings
- [x] feat(svelte): provides workframe content function

### How to test?

1. `npm i`
2. `npm t`
3. `npm -w packages/ui run dev`

It should load Atelier's UI workbench, as a proof of Svelte support.
